share/spack/templates/depfile/Makefile

```diff
@@ -1,4 +1,4 @@
-SPACK ?= spack -c config:install_status:false
+SPACK ?= {{ spack_script }} -c config:install_status:false
```

Comment:
maybe needs to be quoted?

Reply:
You can use `spack_script = shlex.quote(path_to_spack)` on the python side, no need for quotes in the makefile

Reply:
done, thanks